nextjs-nodejs-mongodb/api/guestbook/sign.js

```diff
@@ -0,0 +1,34 @@
+// Import Dependencies
+const connect = require("../../lib/db")
+
+// The main exported function that handles the request and response
+module.exports = async (req, res) => {
+  // Destructure variables from the request body
+  // (converted to JSON using micro)
+  const { signature, id, user } = req.body
+  const updated = new Date()
+
+  // Reject request if no signature is provided in the JSON
+  if (!signature) {
+    res.status(400).json({message: 'Request is missing a "signature" property.'})
```

Comment:
```suggestion
    return res.status(400).json({message: 'Request is missing a "signature" property.'})
```

Don't forget to return here, otherwise the code below will run. An alternative could be to use an `else`.

Reply:
Doesn't .json handle that? I thought .json was a replacement for .end, which should handle `return`, right?

Reply:
It's not really about returning something. It's about not running the code below. Your function should end here and not continue to execute. One solution for this is to `return`, another is to use an `else`.

```js
if() {
 // do a
} else {
 // do b
}
```

is equivalent to:

```js
if() {
 // do a
 return
}
// do b
```